src/mutator_test.cc

```diff
@@ -631,6 +640,25 @@ TEST(MutatorMessagesTest, UsageExample) {
   EXPECT_EQ(3u * 5u, mutations.size());
 }
 
+static void CustomMutation(protobuf::Message* message) {
+  MutatorMessagesTest::custom_mutator_used = true;
+}
+
+TEST(MutatorMessagesTest, FieldMutator) {
+  SmallMessage message;
+  TestMutator mutator(false);
+  TestMutator::RegisterCustomMutation(
+      SmallMessage::descriptor()->FindFieldByName("opt_enum"), CustomMutation);
+
+  for (int j = 0; j < 10000; ++j) {
```

Comment:
you can mutate string field and put some long unique string
then you can check that the string:
1. sometimes that unique string
2. sometimes it's original, unchanged
3. sometimes it's some different string: regular mutator was used.